test/io/postgrest.py

```diff
@@ -104,7 +105,8 @@ def run(
             if wait_for_readiness:
                 wait_until_ready(adminurl + "/ready")
 
-            process.stdout.read()
+            if no_startup_stdout:
+                process.stdout.read()
```

Comment:
Had to add this conditional in order to show the PostgREST startup output to `stdout` (it was empty otherwise).